use_new_columns_from_shared_datasource/README.md

```diff
@@ -0,0 +1,113 @@
+# Tinybird Versions - Use a new column coming from a Shared Data Source
+
+Sharing Data Sources among multiple workspaces is a common practice. One typical scenario involves having a workspace that houses all landing Data Sources, centralizing all ingestion operations, and then sharing them with specific workspaces.
```

Comment:
In case it's useful we have a diagram in this guide => https://www.tinybird.co/docs/guides/how-to-work-with-data-projects.html#how-to-organize-your-data-projects

Reply:
Added that image (that it's way better than the mermaid chart) at e6e056587a9e975cf656f1b7fb14650bedee8ad2